### PR TITLE
added second nft collection to Suricata Rally

### PR DESCRIPTION
**Pull request by ferdinandhodl**

### CODE REVIEW

File: data/suricata-rally.json [hunk 1]
```diff
@@ -27,21 +27,46 @@
...
+        }
+      ]
+    },
+    {
+      "collectionContract": "26Mirs33zojnVMRkqVDJtMZvVZcbAFVyxGojGw7UtWp2K",
+      "collectionLink": "https://alphaga.app/collections/d07ae0d7da7cc282c6f7dec7e1984586147c0d61183936088ca736632ac80500",
+      "collectionName": "Suricata Rally Mascot Static Collection",
+      "collectionPreview": [
+        {
+          "image_url": "/dapps/suricata-rally/suricata-static-nft-collection-Caveman",
```
Comment (Fugashu) [inline data/suricata-rally.json]: Seems like the image url is wrong and misses the extension for this collection.
Also, Alphland right now only has support for one collection

**Review by Fugashu — CHANGES_REQUESTED** [review]
something seems wrong, i guess your images are missing the .png 

Also please only add one collection, multi collection is not supported right now

File: data/suricata-rally.json (hunk 1 above)
Comment (ferdinandhodl) [reply data/suricata-rally.json]: thanks for your review. missing file extension has been fixed

**Comment by ferdinandhodl** [comment]
please pull

**Pull request merged**